source/_integrations/music_assistant.markdown

```
@@ -33,6 +33,132 @@ Under normal circumstances, Home Assistant automatically discovers your running
+### Action `music_assistant.play_media`
+
+Play media on a Music Assistant player with more fine-grained control options.
+
+- **Data attribute**: `media_id`
+  - **Optional**: No.
+  - **Description**: URI or name of the item to be play. Specify a list if it is desired to play/enqueue multiple items.
+  - **Example**: `spotify://playlist/aabbccddeeff`
+- **Data attribute**: `media_type`
+  - **Optional**: Yes.
+  - **Description**: The type of content to play. Select from artist, album, track, playlist or radio. Will be auto-determined if omitted.
+  - **Example**: `playlist` 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)